qa/tasks/cephfs/test_admin.py

```diff
@@ -310,6 +310,15 @@ def test_mds_config_asok(self):
         out = self.fs.mds_asok(['config', 'get', test_key])
         self.assertEqual(out[test_key], test_val)
 
+        # Implicitly asserting that things don't have lockdep error in shutdown
+        self.mount_a.umount_wait(require_clean=True)
+        self.fs.mds_stop()
```

Comment:
Please remove this code kept from a conflict resolution.

Reply:
updated it. Thx